### PR TITLE
Added headless state notifications

### PR DESCRIPTION
**Pull request by Naatan**
https://www.pivotaltracker.com/story/show/176809621

Message is a placeholder, product and Luke should review:

image

### CODE REVIEW

File: locale/en-us.yaml [hunk 1]
```diff
@@ -1641,4 +1641,10 @@ err_activate_platfrom_alternate_branches:
 
     Type [ACTIONABLE]`state branch switch <NAME>`[/RESET] to switch to a different branch.
 err_set_default_branch:
-  other: Could not update project field with branch, please manually set the `branch={{.V0}}` query parameter on the project field in your activestate.yaml.
\ No newline at end of file
+  other: Could not update project field with branch, please manually set the `branch={{.V0}}` query parameter on the project field in your activestate.yaml.
```
Comment (daved) [inline locale/en-us.yaml:1644]: nit:
```suggestion
  other: Could not update project field with branch. Please manually set the `branch={{.V0}}` query parameter on the project field in your activestate.yaml.
```

**Review by daved — APPROVED** [review]

**Comment by daved** [comment]
This PR uses the term "detached", but the URL uses "headless". If these two terms are surely the same, I suggest using only one of them across user-facing content.

**Comment by flukeout** [comment]
Will take a look in the morning tomorrow!

**Comment by Naatan** [comment]
> This PR uses the term "detached", but the URL uses "headless". If these two terms are surely the same, I suggest using only one of them across user-facing content.

Fwiw this is on purpose since so far the internal lingo has been "headless" and to my knowledge we've had no external lingo. Imo "detached" makes more sense for users. Agree it would be nice if internal and external lingo matched.

**Comment by flukeout** [comment]
Here's a take on the copy, I'll get Pete to take a look at it as well. I think we should indicate what a detached state is & indicate why creating a project is useful. Let me know your thoughts, I'll run through it with @rawktron  in our stand as well.

```
Detached State
--------------
Your changes are local and will not be saved the cloud. Visit...
· Visit https://platform.activestate.com/commit/ebb055557-45648-45648-45648 
...to create a project at this point and create a reusable runtime.

For more information, including how to attach to an existing project, visit...
· https://docs.activestate.com/platform/state/advanced-topics/headless
```

**Comment by rawktron** [comment]
> Here's a take on the copy, I'll get Pete to take a look at it as well. I think we should indicate what a detached state is & indicate why creating a project is useful. Let me know your thoughts, I'll run through it with @rawktron in our stand as well.
> 
> ```
> Detached State
> --------------
> Your changes are local and will not be saved the cloud. Visit...
> · Visit https://platform.activestate.com/commit/ebb055557-45648-45648-45648 
> ...to create a project at this point and create a reusable runtime.
> 
> For more information, including how to attach to an existing project, visit...
> · https://docs.activestate.com/platform/state/advanced-topics/headless
> ```
> 
> Not particularly hot on `saved to the cloud`, if you have any ideas to convey that it would be helpful, I'll think some more too.
> 
> cc @Naatan

Yeah, to the cloud is weird. Can we just say "the platform" instead of "the cloud"? You have a double "Visit" in there. Also use a colon and no ellipsis.

Also double `create` in the CTA line is weird. You only need one.

Is there a place to just edit this copy directly?

**Comment by Naatan** [comment]
Here's the latest with Pete's suggestions integrated:

image

**Review by rawktron — APPROVED** [review]

**Pull request merged**